### ISSUE TITLE
[YSQL][LST] FATAL: Check failed: _s.ok() Bad status: Aborted

### ISSUE THREAD

**Issue by def-** [yugabyte/yugabyte-db#13071]
Jira Link: [DB-2791](https://yugabyte.atlassian.net/browse/DB-2791)
### Description

```
$ cd ~/code/yugabyte-db
$ git checkout 206292f0de9d1e04f42ec9fc04c832d1cb0bb34d
$ git revert 85ac8d87d99a214d99f6e701334b3162bfedc7e5.
$ ./yb_build.sh release
$ bin/yb-ctl --replication_factor 1 create --tserver_flags=yb_enable_read_committed_isolation=true --master_flags=yb_enable_read_committed_isolation=true
$ cd ~/code/yb-long-system-test
$ git checkout 88de974e627a6bd26d2e2141d36be488c5322761
$ python3.9 ./long_system_test.py --nodes=127.0.0.1:5433 --threads=10 --runtime=0 --max-columns=10 --seed=032216 --complexity=full
2022-06-27 20:32:04,944 MainThread INFO     Reproduce with: git checkout 88de974e && ./long_system_test.py --nodes=127.0.0.1:5433 --threads=10 --runtime=0 --max-columns=10 --seed=032216 --complexity=full
2022-06-27 20:32:05,606 MainThread INFO     Database version: PostgreSQL 11.2-YB-2.15.1.0-b0 on x86_64-pc-linux-gnu, compiled by clang version 12.0.1 (https://github.com/yugabyte/llvm-project.git bdb147e675d8c87cee72cc1f87c4b82855977d94), 64-bit
2022-06-27 20:32:05,609 MainThread INFO     Creating tables for database db_lst_032216
2022-06-27 20:32:12,450 MainThread INFO     Starting worker_0: RandomSelectAction, SetConfigAction
2022-06-27 20:32:12,458 MainThread INFO     Starting worker_1: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-27 20:32:12,459 MainThread INFO     Starting worker_2: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-27 20:32:12,460 MainThread INFO     Starting worker_3: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-27 20:32:12,461 MainThread INFO     Starting worker_4: RandomSelectAction, SetConfigAction
2022-06-27 20:32:12,468 MainThread INFO     Starting worker_5: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-06-27 20:32:12,472 MainThread INFO     Starting worker_6: RandomSelectAction, SetConfigAction
2022-06-27 20:32:12,472 MainThread INFO     Starting worker_7: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-27 20:32:12,476 MainThread INFO     Starting worker_8: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-06-27 20:32:12,477 MainThread INFO     Starting worker_9: RandomSelectAction, SetConfigAction
2022-06-27 20:32:22,487 MainThread INFO     Worker queries/s: [004.0][000.9][000.6][000.9][004.4][007.1][004.0][000.1][007.4][002.9]
[...]
2022-06-27 22:51:41,950 MainThread INFO     Worker queries/s: [000.1][000.0][000.0][000.0][000.0][000.1][000.0][000.0][000.0][000.0]
2022-06-27 22:51:47,916 worker_0   ERROR    Unexpected query failure: InternalError_
Query: SELECT ('[-62.6034223801009,86.25000437531637)'::NUMRANGE) * ('[-19.464415214562862,98.7696584062164]'::NUMRANGE), -41.738609977523076, ('[41,43]'::INT4RANGE) * (int4range((39), NULL)), '{"a": 7, "b": ["0", "1", "2", "3", "4", "5", "6"], "c": false}'::jsonb, -98.97327829954521, count(t2.c5_daterange), 96.48169765380888, '{"a": 9, "b": ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"], "c": true}'::json, count(*), '[39,)'::INT4RANGE, upper('44'), '1931-04-19', '[-19,58)'::INT2RANGE, 4.554141020729304, (-77) / (58), -78.26541243840276, '-52', 89.16246747951351, '(-41,5)'::INT2RANGE, ('-57') || (66), coalesce(upper(('(,-54]'::INT8RANGE)), ((97) + (-73))), count(t2.c21_float8), '-97', ('(,76]'::INT2RANGE) * ('(-37,1)'::INT2RANGE), 93.79039182660716, '[-90,46]'::INT2RANGE, 18.11106753648899, '1911-04-26', ||/ ((coalesce(upper(('[-48,0)'::INT4RANGE)), (1))) + (-38)), 78.36612024504512, numrange((13.663486232423665), NULL), CAST(((abs(41.78997230587157)) * (-92.7232225276041)) AS INT), '1940-11-07', '(1973-08-12,2000-03-26]'::DATERANGE, (('[37,)'::INT2RANGE) * (('(,-78]'::INT2RANGE) * ('[-45,-1]'::INT2RANGE))) @> ('(-24,)'::INT2RANGE), coalesce(upper((daterange(('1906-11-25'), NULL))), (coalesce(upper(('[1956-04-18,1995-05-26)'::DATERANGE)), (coalesce(upper(((daterange(('1915-01-15'), NULL)) * ('(1903-11-15,1924-05-22)'::DATERANGE))), ('1930-10-20')))))), range_merge(('(-22,-9)'::INT2RANGE), ('[-55,)'::INT2RANGE)), count(t2.c37_date), 59, -2147483648, -4.758111195690276, '{"a": 8, "b": ["0", "1", "2", "3", "4", "5", "6", "7", "8"], "c": false}'::json, TRUE, -46.422454799107385, CAST((76.90547236165938) AS BIGINT), -9, 81.75971969288287, count(t2.c47_numrange), '{"a": 6, "b": ["0", "1"], "c": true}'::jsonb, coalesce(upper((range_merge((range_merge(('(-74,-19]'::INT2RANGE), ('(-20,-16)'::INT2RANGE))), (range_merge((range_merge(('[32767,)'::INT2RANGE), ('(,-100]'::INT2RANGE))), ('(50,)'::INT2RANGE)))))), (coalesce(upper(((('[-70,-62]'::INT2RANGE) * (('(-57,15)'::INT2RANGE) * ('(-92,34)'::INT2RANGE))) * (('[-40,)'::INT2RANGE) * ('(,57]'::INT2RANGE)))), (15)))), '[72,)'::INT4RANGE, '(,4)'::INT4RANGE, 73, (numrange(NULL, (CAST((-31.629649933677825) AS NUMERIC)))) * ('[-42.27515075099364,-19.550596177101596]'::NUMRANGE), 57.82064580828305, '1964-09-23', ('empty'::INT8RANGE) <= ('(,54]'::INT8RANGE), '30', 21.366607699070016, -77.48821875892523, '[-91.55583781068285,9.992232262617009]'::NUMRANGE, '(,-91]'::INT8RANGE, '88', ('(-98,60)'::INT2RANGE) * ('[-8,50)'::INT2RANGE), (daterange(NULL, ('2009-11-17'))) <= (('[2008-07-24,2009-11-10)'::DATERANGE) * ('[1963-09-29,1967-12-21)'::DATERANGE)), '(-58,4)'::INT4RANGE, '(25,60)'::INT8RANGE, int4range((-75), NULL), '{"a": 4, "b": ["0", "1", "2"], "c": false}'::jsonb, -59, FALSE, FALSE, CAST((22) AS BIGINT), '(,-32768]'::INT2RANGE, '2007-03-19', '{"a": 2, "b": ["0", "1", "2", "3", "4"], "c": true}'::jsonb, -96.59027175052064, '{"a": 2, "b": ["0", "1"], "c": true}'::json, coalesce(lower((('[-88.29421652303387,-50.07105881478362]'::NUMRANGE) * ('[73.8798360316836,74.42189361318455)'::NUMRANGE))), (-24.924141254317746)), numrange(NULL, (CAST((-56.12349843165478) AS NUMERIC))), 32.19388932700258, '{"a": 4, "b": ["0", "1", "2", "3"], "c": true}'::json, '(1912-12-13,1968-03-21]'::DATERANGE, TRUE, CAST((-4.379213823104237) AS BIGINT), (range_merge(('[-54.439941046931374,-10.280305782402593)'::NUMRANGE), (numrange((-8.182695345371855), NULL)))) >> ('[-51.81955807291929,15.131794900580616)'::NUMRANGE), '[1924-11-25,1966-02-12]'::DATERANGE, CAST((-15.712963551440453) AS SMALLINT), '[-38,64]'::INT8RANGE, coalesce(upper(('(1911-11-18,1940-10-23]'::DATERANGE)), ('1940-12-26')), 43, '{"a": 4, "b": ["0", "1", "2", "3", "4", "5", "6", "7", "8"], "c": false}'::json, '[-32,36)'::INT4RANGE, (-94) - (-99) FROM t2 WHERE TRUE ORDER BY 21 ASC;
  values: None
  runtime: 2022-06-27 22:51:32.125 - 2022-06-27 22:51:47.916
  supports explain: True
  supports rollback: True
  affected rows: None
Action: RandomSelectAction
Error class: InternalError_
Error code: XX000
Error message: ERROR:  Service unavailable: Shutdown connection
Transaction isolation level: repeatable read
DB Node: host: 127.0.0.1, port: 5433
DB Backend PID: 1281515
```
```
[deen@devp yugabyte-data]$ cat ./node-1/disk-1/yb-data/tserver/logs/yb-tserver.FATAL.details.2022-06-27T20_51_47.pid1281518.txt
F20220627 20:51:47 ../../../../../../src/yb/yql/pggate/pggate.cc:1718] Check failed: _s.ok() Bad status: Aborted (yb/rpc/reactor.cc:94): Shutdown connection (system error 108)
    @     0x7f52fd7a651c  google::LogDestination::LogToSinks()
    @     0x7f52fd7a0c4f  google::LogMessage::SendToLog()
    @     0x7f52fd7a1558  google::LogMessage::Flush()
    @     0x7f52fd7a3e5f  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f5302152378  yb::pggate::PgApiImpl::ClearSeparateDdlTxnMode()
    @           0x9c5ef6  YBTxnDdlProcessUtility
    @           0x8436da  PortalRunUtility
    @           0x842d97  PortalRunMulti
    @           0x842536  PortalRun
    @           0x83fd36  yb_exec_simple_query_impl
    @           0x8402d6  yb_exec_query_wrapper_one_attempt
    @           0x83d358  PostgresMain
    @           0x7ad46c  BackendRun
    @           0x7ac8c0  ServerLoop
    @           0x7a8f2b  PostmasterMain
    @           0x712c5d  PostgresServerProcessMain
    @           0x713102  main
    @     0x7f530172d825  __libc_start_main
    @           0x4b8659  _start
```
I can provide full yugabyte-data directory for debugging.

**Comment by def-** [comment]
Happened again on 226465305d7db859424e06178723b445b6bf964c. Server is in unreachable state afterwards:
```
[deen@devp yugabyte-db]$ bin/ysqlsh
ysqlsh: could not connect to server: Connection refused
	Is the server running on host "localhost" (::1) and accepting
	TCP/IP connections on port 5433?
could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5433?
```

**Comment by fizaaluthra** [comment]
Closing as part of #13128  -- addressed in commit 5a6d54a6995b3d45dbba257abde296e91fa63e54
